lms/templates/fields/field_readonly.underscore

```diff
@@ -1,10 +1,8 @@
-<label class="u-field-title" aria-hidden="true">
-    <%- title %>
-</label>
-<span class="sr" for="u-field-input-<%- id %>"><%- screenReaderTitle %></span>
-<span class="u-field-value">
-    <input id="u-field-input-<%- id %>" aria-describedby="u-field-message-<%- id %>" type="text" name="input" readonly=true value="<%- value %>">
-</span>
+<% if (title) { %>
+<span class="u-field-title" aria-hidden="true"><%- title %></span>
```

Comment:
@cahrens sorry, I was referring to this in the context of AC-200, which I was looking at with @clytwynec yesterday.    Disregard!